Simplenote/Classes/NoteOptionsViewController.swift

```diff
@@ -0,0 +1,289 @@
+//
+//  NoteOptionsViewController.swift
+//  Simplenote
+//
+//  Created by Matthew Cheetham on 11/08/2020.
+//  Copyright © 2020 Automattic. All rights reserved.
+//
```

Comment:
Mind dropping all of the headers? (I still need to go thru the whole project, but by convention, all of our files should be header-less).

Thank you!!

Reply:
No problem